docs/variants/novacustom_ns5x_7x_12th_gen/building-manual.md

```diff
@@ -0,0 +1,57 @@
+# Building manual
+
+## Intro
+
+This document describes the procedure for compiling coreboot for NovaCustom
+NS5xPU/NS7xPU.
+
+## Requirements
+
+- Docker
+    + follow [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+    + follow [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
+- Git
+
+## Procedure
+
+The easiest way to build coreboot is to use the official Docker image.
+
+1. Obtain the image:
+
+    ```bash
+    docker pull coreboot/coreboot-sdk:2021-09-23_b0d87f753c
+    ```
+
+1. Clone the coreboot repository:
+
+    ```bash
+    git clone https://review.coreboot.org/coreboot.git
+    ```
+
+1. Checkout to the desired Dasharo revision:
+
+    > Replace the REVISION with one of the:
+    > - `clevo/release-adl-p` for the latest released version
```

Comment:
This branch doesn't exist

Reply:
Yet

Reply:
Yes I'd say we want that. They all have the same base already AFAIK.